speaking-portal-project/src/types.ts

```diff
@@ -13,17 +13,20 @@ export type MouthCueArray = Array<MouthCue>
 
 export type Phoneme = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H' | 'X'
 
-// TODO: update when more avatars added
+// TODO: when updating avatars, update its name here
 export enum AvatarNames {
     barb,
     boy,
     __LENGTH
 }
 
-// TODO: update when more poses added
+// TODO: when updating body poses, update pose names here
+// TODO: remove temporary poses and add artist poses once received
 enum Pose {
     default,
     relaxed,
+    pos1,
+    pos2,
     __LENGTH
```

Comment:
Does not really apply for this PR...

What is __LENGTH for ? 

And are pos1 and pos1 referring to a particular pose (raised hands, relaxed hands)?

Reply:
The pos1 and pos2 are the temporary poses so that I could check the pose change logic a bit easier. It's difficult to judge pose timings with only two poses. These temporary poses will be replaced once we get the actual poses from the artist

Reply:
`__LENGTH` is used in the `chooseNewPose()` function so that it can randomly choose a new pose that is different from the current one. I needed to be able to index it, and that length value lets me do that